workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/GameObjectWorldCommandSystem.cs

```diff
@@ -0,0 +1,252 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Unity.Collections;
+using Unity.Entities;
+
+#pragma warning disable 649
+
+namespace Improbable.Gdk.Core.GameObjectRepresentation
+{
+    // ReSharper disable once ClassNeverInstantiated.Global
+    public class GameObjectWorldCommandSystem : ComponentSystem
+    {
+        private struct WorldCommandResponderData : ISystemStateComponentData
+        {
+            public uint ResponderHandle;
+
+            public HashSet<WorldCommands.WorldCommandResponseHandler> ResponseHandlers
```

Comment:
If the main operation done on this is to iterate over it then a list might be better.

Reply:
Fixing